### DISCUSSION TITLE
Dropdown/menu items dont have a hover focus

### DISCUSSION THREAD

**Discussion by TreyWW** [saadeghi/daisyui#2572] [Q&A] [answered]
in my code:
![image](https://github.com/saadeghi/daisyui/assets/73353716/905e2912-87ae-4815-b741-e9e213905d8d)

on tailwind play:
![image](https://github.com/saadeghi/daisyui/assets/73353716/415987b7-1ce6-4918-a884-19a9c2a281f5)

I've used the same input.css files, same config files, and all of the same "base" files. But for some reason no dropdown hovers are working. It's quite annoying to be fair, it's just I dont know where to even start looking to debug this, it works completely fine in tailwind play.

I have tried with my full base files like I mentioned, all the way from `<html>` to `<head>` etc. Tried with a default input.css file, but still nothing.

A possible guess is that its something to do with z-index's or something? I'm not sure to be honest, just need a bit of guidance

Thanks

**Comment by saadeghi** [comment]
Are you using the latest version?

**Reply by TreyWW** [reply]
Yep

**Reply by saadeghi** [reply]
Please share a reproduction repo so I can reproduce the issue

**Reply by TreyWW** [reply]
Okay cool, the project it's in is open source so would you like me to link the files from that GitHub, or I could send the tailwind play link that just has base, head, and then the dropdown

**Reply by saadeghi** [reply]
Both are okay. I just need a setup where I can reproduce the issue and see the bug because the issue doesn't exist in daisyUI website nor Tailwind Play so I assume either you're not using the latest version, or the issue is from another CSS in your project.

**Comment by TreyWW** [comment]
Okay cool; so some main files from our project:

- [package.json daisyui v4.4.0](https://github.com/TreyWW/MyFinances/blob/da9c73f5485801139f3eee967ef9eeb2458248f4/package.json#L15C5-L15C25)
- [input.css (still bug without any modifications to this though)](https://github.com/TreyWW/MyFinances/blob/main/frontend/static/src/input.css)
- [head (uses output.css so not much info here)](https://github.com/TreyWW/MyFinances/blob/main/frontend/templates/base/_head.html)
- [topbar.html (has a dropdown)](https://github.com/TreyWW/MyFinances/blob/main/frontend/templates/base/_topbar.html)
- [base.html](https://github.com/TreyWW/MyFinances/blob/main/frontend/templates/base/base.html)

And i put them files I linked in a tailwind play, and for some reason, works just fine. Using same base, head, css
https://play.tailwindcss.com/2QnGZ5W8P6?layout=horizontal

Thanks again for the help

**Reply by TreyWW** [reply]
Also an "npm list" shows
+-- daisyui@4.4.4


Edit: oh crap damn... theres 4.4.6 now.. ill try that 🤦
I only started adding tailwind 2 days ago, wasnt expecting an update

**Reply by saadeghi** [reply, marked as answer]
The issue started from `4.4.3` and was fixed in `4.4.5`   
The latest version is `4.4.6`. this library (like many others) is constantly updating with bug fixes and improvements so always try the latest version.

**Comment by TreyWW** [comment]
Aha works fine now in 4.4.6, you were right. So sorry for that, I didn't even think that there'd be a new version, didn't even check. 

Seems like 4.4.5 and 4.4.6 both fixed bugs related to the menu, which is what my issue was. 

Sorry again, great project though :)
